services/db/recovery.go

```diff
@@ -149,6 +150,24 @@ func alertOps(mode string) {
 	}
 }
 
+// sendRecoveryModeStat sends the recovery mode metric every 1 minute
+func sendRecoveryModeStat() {
+	for {
+		time.Sleep(1 * time.Minute)
+		recoveryModeStat := stats.NewStat("recovery.mode_normal", stats.GaugeType)
+		if CurrentMode != normalMode {
```

Comment:
Why separate condition for normalMode? one switch case for entire thing maybe?

Reply:
Switched to switch/case